oracle/provider/helix.go

```diff
@@ -58,6 +75,26 @@ func NewHelixProvider(
 		nil,
 	)
 
+	res, err := http.Get(helixTokenFile)
+	if err != nil {
+		return nil, err
+	}
+	body, err := io.ReadAll(res.Body)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	var response []HelixToken
+	err = json.Unmarshal(body, &response)
+	if err != nil {
+		return nil, err
+	}
+	tokens := map[string]HelixToken{}
+	for _, helixToken := range response {
+		tokens[strings.ToUpper(helixToken.Denom)] = helixToken
```

Comment:
I noticed that ToUpper is used in several places throughout the code. What do you think about creating a utility function for it to reduce repetition and improve readability?